tests/test_missionsearch.py

```diff
@@ -550,11 +550,11 @@ def test_filter():
 
 def test_tess_clouduris():
     """regression test - do tesscut/nan's in dataURI column break cloud uri fetching"""
-    toi = TESSSearch("TOI 1161", sector=14)
-    # 17 products should be returned
-    assert len(toi.cloud_uris) == 20
-    # 5 of them should have cloud uris
-    assert np.sum((toi.cloud_uris.values != None).astype(int)) == 6
+    toi = TESSSearch("TOI 1161", hlsp=False, sector=14)
+    # 12 products should be returned
+    assert len(toi.cloud_uri) == 12
+    # 6 of them should have cloud uris
```

Comment:
This seems potentially brittle, but we do need to check cloud uri values somehow.